### PR TITLE
Add rotated numpad variant of Keeb.io Viterbi

### PR DESCRIPTION
**Pull request by joshchngs**
## Description

This is just the right-hand side of a Viterbi rev2 used as a numpad with
a few media & special keys.

You could apply the same concept to the left-hand side, but you would have
to use a 1U enter, since the 2U space would be on the wrong side.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [ ] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by drashna** [comment]
Honestly, I'm not sure that this is needed. 

You can add a custom layout, and redefine the matrix values.  I've done exactly that, in fact:
https://github.com/qmk/qmk_firmware/tree/master/keyboards/keebio/viterbi/keymaps/drashna

File: keyboards/keebio/viterbi_numpad/keymaps/default/config.h [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+// #define USE_I2C
+
+/* Select hand configuration */
+
+// #define MASTER_RIGHT
+// #define EE_HANDS
+
+#undef RGBLED_NUM
+#define RGBLIGHT_ANIMATIONS
```
Comment (drashna) [inline keyboards/keebio/viterbi_numpad/keymaps/default/config.h:28]: Could you explicitly define which animations are enabled? 
```suggestion
#define RGBLIGHT_EFFECT_BREATHING
#define RGBLIGHT_EFFECT_RAINBOW_MOOD
#define RGBLIGHT_EFFECT_RAINBOW_SWIRL
#define RGBLIGHT_EFFECT_SNAKE
#define RGBLIGHT_EFFECT_KNIGHT
#define RGBLIGHT_EFFECT_CHRISTMAS
#define RGBLIGHT_EFFECT_STATIC_GRADIENT
#define RGBLIGHT_EFFECT_RGB_TEST
#define RGBLIGHT_EFFECT_ALTERNATING
#define RGBLIGHT_EFFECT_TWINKLE
```

File: keyboards/keebio/viterbi_numpad/keymaps/default/keymap.c [hunk 2]
```diff
@@ -0,0 +1,35 @@
+#include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/keebio/viterbi_numpad/keymaps/default/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

Or 

https://github.com/qmk/qmk_firmware/blob/6b299fab6e6a3143c061b9ea005253361fcd524b/data/templates/base/keyboard.c#L1-L15

File: keyboards/keebio/viterbi_numpad/readme.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+A repurposing of the right half of a regular Viterbi (rotated) for use as a numpad with a few extra keys.
+This is probably not useful for anybody apart from me, but I have a spare right PCB and need a numpad.
+
+I haven't included a rev1 version, since I only have a rev2.
+
+Keyboard Maintainer: [Josh Channings](https://github.com/joshch)
+Hardware Supported: Elite C
+Hardware Availability: [Keebio](https://keeb.io)
+
+Make example for this keyboard (after setting up your build environment):
```
Comment (drashna) [inline keyboards/keebio/viterbi_numpad/readme.md:13]: Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```

File: keyboards/keebio/viterbi_numpad/readme.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
+Viterbi (numpad variant)
+========================
```
Comment (drashna) [inline keyboards/keebio/viterbi_numpad/readme.md:2]:
```suggestion
# Viterbi (numpad variant)
```

File: keyboards/keebio/viterbi_numpad/readme.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
+A repurposing of the right half of a regular Viterbi (rotated) for use as a numpad with a few extra keys.
+This is probably not useful for anybody apart from me, but I have a spare right PCB and need a numpad.
+
+I haven't included a rev1 version, since I only have a rev2.
+
+Keyboard Maintainer: [Josh Channings](https://github.com/joshch)
+Hardware Supported: Elite C
+Hardware Availability: [Keebio](https://keeb.io)
```
Comment (drashna) [inline keyboards/keebio/viterbi_numpad/readme.md:11]: it it really available at keebio?
```suggestion
* Keyboard Maintainer: [Josh Channings](https://github.com/joshch)
* Hardware Supported: Elite C
* Hardware Availability: [Keebio](https://keeb.io)
```

File: keyboards/keebio/viterbi_numpad/rev2/rev2.c [hunk 6]
```diff
@@ -0,0 +1,18 @@
+#include "viterbi.h"
```
Comment (drashna) [inline keyboards/keebio/viterbi_numpad/rev2/rev2.c:1]: Could you add a GPL2+ compatible license header here? 

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

Or 

https://github.com/qmk/qmk_firmware/blob/6b299fab6e6a3143c061b9ea005253361fcd524b/data/templates/base/keyboard.c#L1-L15

**Review by drashna — CHANGES_REQUESTED** [review]

**Comment by joshchngs** [comment]
Sory @drashna, I thoughty I closed this.

Your first suggestion of using a custom layout is probably what I needed. I'll close this and resubmit with the correct version next time I make a change to it.

For the rest of your comments - I just duplicated the standard keyboard definition. I haven't tested animations or anything like that. Again, probably another sign that this is just a custom layout.

Thanks for your time.

**Pull request closed without merging**